### PR TITLE
Catalog and Instance UI to support Headless CMS

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- add fetching mechanism for docs from docsTopics in Catalog and Instances UI, which is default mechanism of presenting docs
- existing solution will be fallback, using deprecated apis

**Related issue(s)**

See #713

### CODE REVIEW

File: catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js [hunk 1]
```diff
@@ -1,104 +1,330 @@
...
-                theme={asyncApiTheme}
-                config={asyncApiConfig}
-              />
+
+      const deprecatedDocs =
+        documentsTypes &&
+        documentsTypes.map(type =>
+          documentsByType &&
+          documentsByType[type] &&
+          validatDocumentsByType(documentsByType[type]) ? (
```
Comment (akucharska) [inline catalog/src/components/ServiceClassDetails/ServiceClassTabs/ServiceClassTabs.component.js]: if(!...){ 
return null;
}
return  <Tab ...>

File: components/react/src/components/ReactMarkdown/components/Theme.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
+export default {
+  plain: {
+    backgroundColor: 'rgb(250, 250, 250)',
+    color: 'rgb(11, 116, 222)',
+  },
+  styles: [
+    {
+      types: ['string'],
```
Comment (akucharska) [inline components/react/src/components/ReactMarkdown/components/Theme.js]: This is older version of file

**Comment by aerfio** [comment]
One thing I want to discuss is whether to leave `console.error` in 
```
   if (error) {
      console.error(error);
      return <div>{`${error.name}: ${error.message}`}</div>;
    }
```

**Comment by aerfio** [comment]
both instances and catalog-ui

**Pull request merged**